hardhat.config.zksync.ts

```diff
@@ -0,0 +1,157 @@
+
+extendConfig((config: HardhatConfig) => {
+  if (process.env.EXPORT !== "true") {
+    config.external = {
+      ...config.external,
+      deployments: {
+        zksyncsepolia: [],
+      },
+    };
+  }
+});
+
+const config: HardhatUserConfig = {
+  defaultNetwork: "hardhat",
+  zksolc: {
+    version: "1.5.1",
+    compilerSource: "binary",
+    settings: {
+      metadata: {
+        // do not include the metadata hash, since this is machine dependent
+        // and we want all generated code to be deterministic
+        // https://docs.soliditylang.org/en/v0.7.6/metadata.html
+        bytecodeHash: "none",
+      },
+    },
+  },
+  solidity: {
+    compilers: [
+      {
+        version: "0.5.16",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 200,
+          },
+          outputSelection: {
+            "*": {
+              "*": ["storageLayout"],
+            },
+          },
+        },
+      },
+      {
+        version: "0.8.25",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 10000,
+          },
+          evmVersion: "paris",
+          outputSelection: {
+            "*": {
+              "*": ["storageLayout"],
+            },
+          },
+        },
+      },
+    ],
+  },
+  zksolc: {
+    version: "1.5.0",
+    settings: {},
+  },
+  networks: {
+    hardhat: isFork(),
+    zksyncsepolia: {
+      url: process.env.ARCHIVE_NODE_zksyncsepolia || "https://sepolia.era.zksync.dev",
+      ethNetwork: "sepolia",
+      verifyURL: "https://explorer.sepolia.era.zksync.dev/contract_verification",
+      accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
+      zksync: true,
+      live: true,
+    },
+  },
+  paths: {
+    sources: "./contracts",
+    tests: "./tests/hardhat",
+    cache: "./cache-zk",
+    artifacts: "./artifacts-zk",
+  },
+  mocha: {
+    timeout: 200000000,
+  },
+  typechain: {
+    outDir: "typechain",
+    target: "ethers-v5",
+  },
+  // Hardhat deploy
+  namedAccounts: {
+    deployer: {
+      default: 0, // here this will by default take the first account as deployer
+    },
+  },
+  external: {
+    contracts: [
+      {
+        artifacts: "node_modules/@venusprotocol/governance-contracts/artifacts-zk",
+      },
+      {
+        artifacts: "node_modules/@venusprotocol/oracle/artifacts-zk",
+      },
+      {
+        artifacts: "node_modules/@venusprotocol/protocol-reserve/artifacts-zk",
+      },
+    ],
+    deployments: {},
+  },
+  docgen: {
+    outputDir: "./docgen-docs",
+    pages: "files",
+    templates: "docgen-templates",
+  },
+};
+
+function isFork() {
+  return process.env.FORK === "true"
+    ? {
+        allowUnlimitedContractSize: false,
+        loggingEnabled: false,
+        forking: {
+          url: process.env[`ARCHIVE_NODE_${process.env.FORKED_NETWORK}`] || "https://sepolia.era.zksync.dev",
+          blockNumber: 21068448, // TODO: change this
```

Comment:
Is there any change to be done here?
```suggestion
          blockNumber: 21068448,
```

Reply:
Yes, block number needs to be updated.
Done [a4061b3](https://github.com/VenusProtocol/venus-protocol/pull/500/commits/a4061b3c612ea137697707f396388ad7551e6ff6)